include/apps/tools/OswAppCalculator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+
+class OswAppCalculator : public OswApp {
+  public:
+    OswAppCalculator(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override; 
+    virtual void loop() override; //checks the cursor position (iNum) and starts the appropriate function
+    virtual void stop() override;
+    void draw(); //draws the ui
+    void setPNum1(); //gets the input that tells if num1 is positive or negative
+    void setNum1(int num1[]); //is used to fill the array that will become num1
```

Comment:
Can you combine two functions that have similar functions into one? in a way that carries only parameters.
`setNum1()` and `setNum2()`